flink-core/src/main/java/org/apache/flink/api/common/operators/base/PartitionOperatorBase.java

```diff
@@ -51,12 +53,19 @@
 	private Partitioner<?> customPartitioner;
 	
 	private DataDistribution distribution;
+
+	private Ordering ordering;
 	
 	
 	public PartitionOperatorBase(UnaryOperatorInformation<IN, IN> operatorInfo, PartitionMethod pMethod, int[] keys, String name) {
 		super(new UserCodeObjectWrapper<NoOpFunction>(new NoOpFunction()), operatorInfo, keys, name);
 		this.partitionMethod = pMethod;
 	}
+
+	public PartitionOperatorBase(UnaryOperatorInformation<IN, IN> operatorInfo, PartitionMethod pMethod, int[] keys, Order[] orders, String name) {
```

Comment:
I think this constructor is not used. So it can be removed. 
Also `orders` is not set in this constructor.